cli/src/commands/next.rs

```diff
@@ -74,8 +84,6 @@ pub(crate) fn cmd_next(
         ui,
         &mut workspace_command,
         &Direction::Next,
```

Comment:
nit: pass `Direction::Next` by value (applies to all callers)

Reply:
Lol... I'd forgotten I'd derived Copy. Thanks. Fixed.